admin_guide/limits.adoc

```diff
@@ -11,36 +11,21 @@
 
 toc::[]
 
-== Overview
+== Purpose for limit ranges
 
 // tag::admin_limits_overview[]
-A limit range, defined by a `*LimitRange*` object, enumerates
-xref:../dev_guide/compute_resources.adoc#dev-compute-resources[compute resource
-constraints] in a xref:../dev_guide/projects.adoc#dev-guide-projects[project] at the pod,
-container, image, image stream, and persistent volume claim level, and specifies the amount of resources
-that a pod, container, image, image stream, or persistent volume claim can consume.
+A limit range, defined by a `*LimitRange*` object, enumerates xref:../dev_guide/compute_resources.adoc#dev-compute-resources[compute resource constraints] in a xref:../dev_guide/projects.adoc#dev-guide-projects[project] at the pod, container, image, image stream, and persistent volume claim level, and specifies the amount of resources that a pod, container, image, image stream, or persistent volume claim can consume.
 
-All resource create and modification requests are evaluated against each
-`*LimitRange*` object in the project. If the resource violates any of the
-enumerated constraints, then the resource is rejected. If the resource does not
-set an explicit value, and if the constraint supports a default value, then the
-default value is applied to the resource.
-
-For CPU and Memory limits, if you specify a `max` value, but do not specify a
-`min` limit, the resource can consume CPU/memory resources greater than `max` value`.
+All resource create and modification requests are evaluated against each `*LimitRange*` object in the project. If the resource violates any of the enumerated constraints, then the resource is rejected. If the resource does not set an explicit value, and if the constraint supports a default value, then the default value is applied to the resource.
```

Comment:
s/constraints, then the resource/constraints, the resource 
ISG strikes again about not needing "then" in most "if/then" sentences. I think an argument can be made that it is helpful to leave it in the last sentence, though.